coders/composio_coders/cli.py

```diff
@@ -0,0 +1,147 @@
+        if curr_repo_name:
+            click.echo(f"no git repo-given. Initializing git repo from current directory: {curr_repo_name}")
+            repo_name = curr_repo_name
+        else:
+            click.echo("❗!! Error: no git repo found or given. Exiting setup...")
+            return
+    issue_id = click.prompt("Please enter issue id", type=str)
+    base_commit_id = click.prompt("Please enter base commit id", type=str, default="", show_default=False)
+    issue_description = click.prompt("Please enter issue description", type=str)
+    issue_config = {
+        'repo_name': repo_name,
+        'issue_id': issue_id,
+        'base_commit_id': base_commit_id,
+        'issue_description': issue_description
+    }
+    config_path = Path(ISSUE_CONFIG_PATH)
+    with config_path.open('w') as f:
+        json.dump(issue_config, f)
+    click.echo(f'🍀 Issue configuration saved to {ISSUE_CONFIG_PATH}\n')
+
+
+@click.command(name="solve", help="👷 Start solving the configured issue")
+def solve():
+    """Start solving the configured issue."""
+    git_access_token = os.environ.get(KEY_GIT_ACCESS_TOKEN)
+    if not git_access_token or not git_access_token.strip():
+        click.echo(f"❗ Error: {KEY_GIT_ACCESS_TOKEN} is not set in the environment.\n")
+        click.echo(f"🔑 Please export your GIT access token: {KEY_GIT_ACCESS_TOKEN} and try again !")
+        return
+
+    config_path = Path(ISSUE_CONFIG_PATH)
+    with config_path.open('r') as f:
+        issue_config = json.load(f)
+
+    click.echo(f"ℹ️ Starting issue solving with the following configuration: {json.dumps(issue_config, indent=4)}")
+
+    args = CoderAgentArgs(
+        repo_name=issue_config['repo_name'],
+        agent_output_dir="./",
+        issue_config={
+            "issue_id": issue_config["issue_id"],
+            "base_commit_id": issue_config["base_commit_id"],
+            "issue_description": issue_config["issue_description"],
+        }
+    )
+    coder_agent = CoderAgent(args)
+    coder_agent.run()
+    print("Issue solving process started.")
+
+
+@click.command(name="reset", help="🔄 Reset the composio coder")
+@click.help_option("--help", "-h", "-help")
+def reset():
+    """Reset the composio coder."""
+    confirmation = click.prompt("Are you sure you want to reset the composio coder? Type 'reset' to confirm or 'cancel' to abort", default="cancel", show_default=False)
+    if confirmation.lower().strip() == "reset":
+        click.echo("Resetting composio coder...")
+        click.echo(f"Removing {MODEL_ENV_CONFIG_PATH} and {ISSUE_CONFIG_PATH}_config files...")
+        if os.path.exists(MODEL_ENV_CONFIG_PATH):
+            os.remove(MODEL_ENV_CONFIG_PATH)
+        if os.path.exists(ISSUE_CONFIG_PATH):
+            os.remove(ISSUE_CONFIG_PATH)
+        click.echo("Composio coder reset complete.")
+    else:
+        click.echo("Reset cancelled.")
+
+
+@click.command(name="workflow", help="📋 Run the workflow: setup -> add_issue -> solve -> reset")
```

Comment:
I think this is a misleading command, it says `Run the workflow` but currently it only print the steps a user needs to perform for the agent to run. Are we planning on adding logic to run the entire workflow via this command?

Reply:
actually i was looking a way to add a banner in the composio-coder cmd, but this was just a hack to show how to run the workflow. I will make the prompt more clearer